### PR TITLE
Add Phone Number detecting

### PR DESCRIPTION
**Pull request by seapy**
relate issue #66 

Regexp is not perfect, but affordable
but if you implement #64 it's good

### CODE REVIEW

File: ActiveLabelTests/ActiveTypeTests.swift [hunk 1]
```diff
@@ -205,6 +210,36 @@ class ActiveTypeTests: XCTestCase {
...
+        XCTAssertEqual(activeElements.count, 1)
+        XCTAssertEqual(currentElementString, "+1-111-1111-1111")
+        XCTAssertEqual(currentElementType, ActiveType.PhoneNumber)
+        
+        label.text = "(111) 111-1111"
+        XCTAssertEqual(activeElements.count, 1)
+        XCTAssertEqual(currentElementString, "(111) 111-1111")
+        XCTAssertEqual(currentElementType, ActiveType.PhoneNumber)
+        
+        label.text = "123456789"
```
Comment (poolqf) [inline ActiveLabelTests/ActiveTypeTests.swift]: For Spain users this one should be valid. I assume the phone regex should be more generic, but don't know how to do it...

**Comment by poolqf** [comment]
Hi @seapy , thanks for the PR, I think it is useful to add detection for phone numbers, but I am not sure how to handle the regex parsing. Depending on the country, there are a lot of different types of phone formats

**Comment by seapy** [comment]
Phone number format has a lot of case. i think regex can't solve all of phone number format.
how about open regex parser. discussed #66 it will helpful.

**Comment by poolqf** [comment]
Hi @seapy , I'm temporary closing this PR until we reach an agreement in the Regex. Let's follow the discusion in #66

**Comment by brunomunizaf** [comment]
I need this so bad rn. @seapy lmk if you found a workaround, in the meantime i'll fork your repo and try to localize the regex for each country that matters for my project at least. Won't make a PR cause I believe that's far from optimal.

**Pull request closed without merging**